app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -36,7 +36,7 @@ class AdministratorControlsViewModel @Inject constructor(
   private val showLogoutDialogListener = activity as ShowLogoutDialogListener
   private lateinit var userProfileId: ProfileId
 
-  /** Sets the Index for the currently selected fragment. */
+  /** Sets index for the currently selected fragment. */
```

Comment:
```suggestion
  /** Sets the index for the currently selected fragment. */
```

That being said, this is still a bit confusing. What exactly is a fragment index?